src/Riok.Mapperly/Riok.Mapperly.csproj

```diff
@@ -1,68 +1,68 @@
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <PackageProjectUrl>https://mapperly.riok.app</PackageProjectUrl>
-    <PackageId>Riok.Mapperly</PackageId>
-    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <PackageDescription>
-      A .NET source generator for generating object mappings.
-      No runtime reflection.
-    </PackageDescription>
-    <PackageTags>Mapper SourceGenerator</PackageTags>
-    <PackageIcon>logo.png</PackageIcon>
-
-    <!-- default roslyn version -->
-    <ROSLYN_VERSION Condition="'$(ROSLYN_VERSION)' == ''">4.5</ROSLYN_VERSION>
-  </PropertyGroup>
-
-  <Target Name="Husky" BeforeTargets="Restore;CollectPackageReferences" Condition="'$(HUSKY)' != 0 AND '$(CI)' == ''">
-    <Exec Command="dotnet tool restore" StandardOutputImportance="Low" StandardErrorImportance="High" />
-    <Exec Command="dotnet husky install" StandardOutputImportance="Low" StandardErrorImportance="High" WorkingDirectory="../.." />
-  </Target>
-
-  <Import Project="Riok.Mapperly.Roslyn$(ROSLYN_VERSION).props" />
-
-  <ItemGroup>
-    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\BannedSymbols.txt" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.4" PrivateAssets="all" >
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>analyzers</IncludeAssets>
-    </PackageReference>
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\Riok.Mapperly.Abstractions\Riok.Mapperly.Abstractions.csproj" PrivateAssets="all" />
-  </ItemGroup>
-
-  <ItemGroup Condition="!$(DefineConstants.Contains('DEBUG_SOURCE_GENERATOR'))">
-    <Compile Remove="Helpers/DebuggerUtil.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <EmbeddedResource Include="..\Riok.Mapperly.Templates\*.cs"
-                      CopyToOutputDirectory="Never"
-                      Visible="false"
-                      LogicalName="Templates/%(FileName)" />
-  </ItemGroup>
-
-  <!--
-  This ensures the library will be packaged as a source generator when we use `dotnet pack`
-  -->
-  <ItemGroup>
-    <None Include="../../README.md" Pack="true" PackagePath="\" />
-    <None Include="../../docs/static/img/logo.png" Pack="true" PackagePath="\" />
-    <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/roslyn$(ROSLYN_VERSION)/dotnet/cs" Visible="false" />
-    <None Include="$(OutputPath)\$(AssemblyName).Abstractions.dll" Pack="true" PackagePath="analyzers/roslyn$(ROSLYN_VERSION)/dotnet/cs" Visible="false" />
-    <None Include="$(OutputPath)\$(AssemblyName).Abstractions.dll" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
-    <None Include="$(OutputPath)\$(AssemblyName).Abstractions.xml" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
-  </ItemGroup>
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
A lot of diffs here... Not sure what changed, but probably revert all except for the expected changes? Probably a change in the file encoding or similar 🤔